test/components/FileInput.spec.js

```diff
@@ -144,4 +150,32 @@ describe('components', () => {
       expect(tree).toMatchSnapshot();
     });
   });
+
+  describe('Metadata', () => {
+    const data = {
+      name: 'Cute puppies',
+      size: 1000,
+    };
+
+    it('should render metadata from default component', () => {
+      const { fileInput } = setup();
+      fileInput.setState({ value: data });
+
+      expect(fileInput.find('FileInputMetadata').length).toEqual(1);
+    });
+
+    it('should render metadata from custom component', () => {
+      const { fileInput } = setup({ customMetadata: CustomComponent });
+      fileInput.setState({ value: data });
+
+      expect(fileInput.find('div').last().hasClass('test')).toBeTruthy();
```

Comment:
You could just `fileInput.find(CustomComponent)`. Imagine that in another user story we change this `CustomComponent` to add something to our tests, or somehow we add another `div` (that will be the last in the `DOM`). The best way to see if it rendered is just to look for itself 😃

Reply:
Such an obvious solution :D Thanks!